plugin-azure-server/src/main/kotlin/jetbrains/buildServer/clouds/azure/arm/AzureCloudImage.kt

```diff
@@ -88,25 +96,30 @@ class AzureCloudImage(private val myImageDetails: AzureCloudImageDetails,
     }
 
     override fun canStartNewInstance(): Boolean {
-        if (activeInstances.size >= myImageDetails.maxInstances) return false
+        if (activeInstances.size >= myImageDetails.maxInstances) {
```

Comment:
if you max out instances the log will be flooded with these WARNs, let's not log anything here